### PR TITLE
add luxon for brief comparison

### PR DESCRIPTION
**Pull request by cylim**
For the detailed comparison, need to understand how we should show for the output: #55 

This merge request is requested by #26

### CODE REVIEW

**Comment by cylim** [comment]
Test failed due to it is using a different alpha version of `date-fns`, should we fix the version until the official release?

**Comment by cht8687** [comment]
@cylim We are using 2.X of date-fns.

**Comment by cylim** [comment]
@cht8687 currently, `date-fns:2.x` is in alpha release, that's the reason for the test fail, do you recommend to fix a version or to fix the unit test along the journey?

**Comment by cylim** [comment]
@cht8687 currently, `date-fns:2.x` is in alpha release, that's the reason for the test fail, do you recommend to fix a version or to fix the unit test along the journey?

**Comment by cylim** [comment]
screenshot 2018-10-22 at 21 46 41

Doesn't sure what is the best way to test this. According to [date-fns docs](https://date-fns.org/v2.0.0-alpha.6/docs/formatDistance), the way they format it is `1 month`, `1 day`, `1 year` instead of `a month`, `a day`, `a year` (momentjs is using).

**Comment by cylim** [comment]
@cht8687 fix the relative time to 3 months. It is not a bug for the test case, but a different library decided to present the relative time in different ways.

### date-fns
1 month, 1 day, 1 year

### moment, dayjs
a month, a day, a year

File: package.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
     "pretty-quick": "^1.6.0"
   },
   "dependencies": {
-    "kebab-case": "^1.0.0"
+    "kebab-case": "^1.0.0",
+    "luxon": "^1.4.4"
```
[the remark this answers is not in the document]
Comment (cylim) [reply package.json]: removed

**Pull request merged**